src/main/groovy/com/jakewharton/sdkmanager/internal/PackageResolver.groovy

```diff
@@ -143,15 +156,11 @@ class PackageResolver {
 
     log.debug "Found support library dependencies: $supportDeps"
 
-    project.repositories.maven {
-      url = androidRepositoryDir
-    }
-
     def needsDownload = false;
     if (!folderExists(androidRepositoryDir)) {
       needsDownload = true
       log.lifecycle 'Support library repository missing. Downloading...'
-    } else if (!dependenciesAvailable(supportDeps)) {
+    } else if (supportDeps.count { it instanceof UnresolvedDependencyResult } > 0) {
```

Comment:
We can trust gradle resolving mechanism that if it thinks that dependency is resolved, that we are all set. No need to make additional check.